core/bones/date.py

```diff
@@ -33,21 +32,26 @@ def __init__(
         """
         super().__init__(**kwargs)
 
-        if creationMagic or updateMagic:
-            self.readonly = True
-
-        self.creationMagic = creationMagic
-        self.updateMagic = updateMagic
-
+        # Either date or time must be set
         if not (date or time):
-            raise ValueError("Attempt to create an empty datebone! Set date or time to True!")
+            raise ValueError("Attempt to create an empty DateBone! Set date or time to True!")
 
+        # Localize-flag only possible with date and time
         if localize and not (date and time):
             raise ValueError("Localization is only possible with date and time!")
+        # Default localize all DateBones, if not explicitly defined
+        elif localize is None and date and time:
+            localize = True
```

Comment:
```suggestion
        elif localize is None:
            localize = date and time
```
otherwise localize would keep `None`.